week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -7,26 +7,31 @@
 
   NOTE: don't include any element that is not a "family".
 */
+function funcMap(index) {
+  return index.indexOf(' ') !== -1 && index[0] === 'A';
```

Comment:
Well done! This does the job.

I would encourage you to look into ways you can make your code easier to read - it's a very useful skill when working in a team.

If you Google the specific task you're trying to achieve (e.g. *check if a word is in a string javascript*) you will often find convenient functions that help you do your task, but may bit easier to use or read, than what you have in mind.

As an example, in this case, you could achieve the same result using the well-named functions:
* `includes`, which can tell you if a string contains a certain word, and
* `startsWith`, which can tell you if a string starts with a certain letter

```javascript
function isGoodFamilyName(name) {
    return name.includes("family") && name.startsWith("A");
}
```